core/modules/system/css/system.admin.css

```diff
@@ -90,6 +99,11 @@ span.admin-enabled {
 span.admin-missing {
   color: #f00;
 }
+@media screen and (max-width: 800px) {
+  #system-modules td.module {
+    width: 110px;
+  }
+}
```

Comment:
@quicksketch Hmm, I was just copying from the other media queries that hide certain columns on smaller screens: https://github.com/backdrop/backdrop/blob/1.x/core/themes/seven/style.css#L476
Might need to create an issue for updating them too.